LockMBeanTest.testMBeanHasLeaseTime_whenLockedWithLeaseTime

```
java.lang.AssertionError: Lease time: 1000
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.assertTrue(Assert.java:41)
    at com.hazelcast.jmx.LockMBeanTest.testMBeanHasLeaseTime_whenLockedWithLeaseTime(LockMBeanTest.java:51)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-pr-builder/com.hazelcast$hazelcast/10521/testReport/junit/com.hazelcast.jmx/LockMBeanTest/testMBeanHasLeaseTime_whenLockedWithLeaseTime/

http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/610

```
15:33:37 [ERROR] testMBeanHasLeaseTime_whenLockedWithLeaseTime_mustHaveRemainingLeaseBeforeItExpires(com.hazelcast.internal.jmx.LockMBeanTest)  Time elapsed: 0.132 s  <<< FAILURE!
15:33:37 java.lang.AssertionError
15:33:37 	at org.junit.Assert.fail(Assert.java:86)
15:33:37 	at org.junit.Assert.assertTrue(Assert.java:41)
15:33:37 	at org.junit.Assert.assertTrue(Assert.java:52)
15:33:37 	at com.hazelcast.internal.jmx.LockMBeanTest.testMBeanHasLeaseTime_whenLockedWithLeaseTime_mustHaveRemainingLeaseBeforeItExpires(LockMBeanTest.java:112)
15:33:37 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:33:37 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
15:33:37 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:33:37 	at java.lang.reflect.Method.invoke(Method.java:498)
15:33:37 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
15:33:37 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
15:33:37 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
15:33:37 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
15:33:37 	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
15:33:37 	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:98)
15:33:37 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
15:33:37 	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast/src/test/java/com/hazelcast/internal/jmx/LockMBeanTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/jmx/LockMBeanTest.java
@@ -102,14 +102,14 @@ public class LockMBeanTest extends HazelcastTestSupport {
 
     @Test
     public void testMBeanHasLeaseTime_whenLockedWithLeaseTime_mustHaveRemainingLeaseBeforeItExpires() throws Exception {
-        lock.lock(1000, TimeUnit.MILLISECONDS);
+        lock.lock(10000, TimeUnit.MILLISECONDS);
         long startTime = Clock.currentTimeMillis();
 
         long remainingLeaseTime = getLongAttribute("remainingLeaseTime");
         long timePassed = Clock.currentTimeMillis() - startTime;
         boolean hasLeaseRemaining = remainingLeaseTime > 0;
 
-        assertTrue(hasLeaseRemaining || timePassed >= 1000);
+        assertTrue(hasLeaseRemaining || timePassed >= 10000);
     }
 
     @Test

